openlibrary/core/fulltext.py

```diff
@@ -49,8 +49,8 @@ def fulltext_search(q, page=1, limit=100):
         for ed in editions:
             if ed.ocaid in ocaids:
                 idx = ocaids.index(ed.ocaid)
-                ia_results['hits']['hits'][idx]['edition'] = ed
+                ia_results['hits']['hits'][idx]['edition'] = \
+                    format_book_data(ed) if js else ed
```

Comment:
```suggestion
                ia_results['hits']['hits'][idx]['edition'] = (
                    format_book_data(ed) if js else ed
                )
```
Let's avoid backslashes in Python code as discussed in PEP8.